### PR TITLE
README: add instructions for kubernetes

### PR DESCRIPTION
**Pull request by philips**
Add a quick and easy way to run this on top of kubernetes. It isn't
production ready but gives people a starting point.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,9 +44,22 @@ Clair detects some vulnerabilities and sends a webhook to your continuous deploy
 During the first run, Clair will bootstrap its database with vulnerability data from its data sources.
 It can take several minutes before the database has been fully populated.
 
+### Kubernetes
+
+An easy way to run clair is with Kubernetes.
+If you are using the [CoreOS Kubernetes single-node instructions](https://coreos.com/kubernetes/docs/latest/kubernetes-on-vagrant-single.html) for vagrant you will be able to access clair at http://172.17.4.99:30061/ after following these instructions.
```
Comment (jzelinskie) [inline README.md]: Can you use the same pattern we're using for URLs?

```
paragraph [link]

[link]: url
```

**Comment by jzelinskie** [comment]
While we're in here, we should probably clarify that both docker compose and k8s instructions are not meant for production, but testing.

**Comment by philips** [comment]
@jzelinskie Done.

File: README.md [hunk 2]
```diff
@@ -44,9 +44,25 @@ Clair detects some vulnerabilities and sends a webhook to your continuous deploy
 During the first run, Clair will bootstrap its database with vulnerability data from its data sources.
 It can take several minutes before the database has been fully populated.
 
+**NOTE:** These setups are not meant for production workloads, but as a quick way to get started.
+
+### Kubernetes
+
+An easy way to run clair is with Kubernetes.
```
Comment (jzelinskie) [inline README.md]: s/clair/Clair

Comment (joshix) [inline README.md]: s/clair/Clair/

**Pull request merged**